tests/patterns/rust/implicit_return.rs

```diff
@@ -0,0 +1,61 @@
+fn f() {
+  // ERROR:
+  let g = || "implicit return for lambda";
```

Comment:
What does this mean in Rust?

Reply:
It creates a lambda that has no arguments.